app/controllers/posts_controller.rb

```diff
@@ -1,17 +1,48 @@
 class PostsController < ApplicationController
+  # load_and_authorize_resource
```

Comment:
- [ ] Please make sure that you have implemented the `authorization` properly, as your previous code reviewer mentioned and suggested you use the `load_and_authorize_resource` callback, to automatically apply the `authorization`, corresponding to the user post in the `post` controller. Kindly uncomment the `callback` and make sure that your application is working as expected.

Reply:
well resolved! ✔️